logic/src/commonMain/kotlin/com/wire/kalium/logic/sync/incremental/EventProcessor.kt

```diff
@@ -51,21 +51,31 @@ internal class EventProcessorImpl(
     private val userPropertiesEventReceiver: UserPropertiesEventReceiver,
 ) : EventProcessor {
 
-    private val logger by lazy { kaliumLogger.withFeatureId(EVENT_RECEIVER) }
+    private val logger by lazy {
+        kaliumLogger.withFeatureId(EVENT_RECEIVER)
+    }
 
     override suspend fun processEvent(event: Event) {
-        logger.i("Processing event ${event.id.obfuscateId()}")
         when (event) {
             is Event.Conversation -> conversationEventReceiver.onEvent(event)
             is Event.User -> userEventReceiver.onEvent(event)
             is Event.FeatureConfig -> featureConfigEventReceiver.onEvent(event)
-            is Event.Unknown -> logger.i("Unhandled event id=${event.id.obfuscateId()}")
+            is Event.Unknown -> {
+                // DO NOTHING
+            }
```

Comment:
Considering we already handle most of the events, wouldn't it be better to keep logging the `Unknown` ones and observe spikes?

Reply:
I think this is logged further down. Let me check

Reply:
Done